executable/elf.ksy

```diff
@@ -703,23 +703,23 @@ types:
     instances:
       program_headers:
         pos: program_header_offset
-        size: program_header_entry_size
+        size: len_program_headers
         type: program_header
         repeat: expr
-        repeat-expr: qty_program_header
+        repeat-expr: num_program_headers
       section_headers:
         pos: section_header_offset
-        size: section_header_entry_size
+        size: len_section_headers
         type: section_header
         repeat: expr
```

Comment:
This renaming is actually misleading - the old `section_header_entry_size` correctly suggested that the field specifies the size of **one** entry, whereas the new `len_section_headers` name incorrectly indicates that it's the total size of all entries, which is not true.

Note that the compiler naming suggestions are pretty crude, taking none of these semantics into account; they can also easily be contradictory, e.g. if you have a `num_*` field that you use in two different `repeat-expr`s. So we always have to think about whether the suggested names make sense.